learn/advanced/asynchronous_updates.md

```diff
@@ -26,34 +37,62 @@ sequenceDiagram
   M->>-Q: dequeue update 2
 </mermaid>
 
-### Which operations are async?
+### Dumps
 
-Every operation which could be compute-expensive is asynchronous. These include:
+While dumps and updates are both asynchronous processes, they use separate queues and behave differently. For instance, creating a new dump will freeze the update queue until the dump has been generated.
 
-- Update index settings
-- Add/update/delete documents
+[You can read more about dumps in our dedicated guide.](/reference/features/dumps.md)
 
 ## Understanding updates
 
-All updates return the following fields:
+After you have requested an update, you can use the [update API endpoint](/reference/api/updates.md) to find out the status of your request. To do so, you will need your request's `updateId`.
+
+### Response
+
+The response you get from the [update API endpoint](/reference/api/updates.md) will always include the following fields:
+
+- `status`: the state of the operation (`enqueued`, `processing`, `processed`, or `failed`)
+- `updateId`: the id of the update
+- `type`: the type of the operation, including its name and number
+- `enqueuedAt`: the date when the operation was added to the queue
 
-- **`status`**: The state of the operation (enqueued, processed, or failed).
-- **`updateId`**: The id of the update.
-- **`type`**: The type of the operation, including its name and number.
-- **`enqueuedAt`**: The date at which the operation has been added to the queue.
+Updates marked as `processed` return additional fields:
 
-Updates that have already been processed also return the following fields:
+- `duration`: the number of seconds taken to complete the operation
+- `processedAt`: the date when the operation was processed
 
-- **`duration`**: The number of seconds taken to complete the operation.
-- **`processedAt`**: The date at which the operation has been processed.
+Finally, failed updates contain an extra `error` field:
 
-Finally, failed updates return an additional field:
+- `error`: a string describing [why the update failed](https://docs.meilisearch.com/errors/)
```

Comment:
In case of an error all error fields are returned inside the returned body: 
```json
{
  "status": "failed", 
  //...
  "message": "Index movies not found",
  "errorCode": "index_not_found",
  "errorType": "invalid_request_error",
  "errorLink": "https://docs.meilisearch.com/errors#index_not_found"
}
```